src/Main.java

```diff
@@ -1,5 +1,67 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        // Задача 1
+        var dog = 8;
+        var cat = 3.6;
+        var paper = 763789;
+        System.out.println("dog = " + dog);
+        System.out.println("cat = " + cat);
+        System.out.println("paper = " + paper);
+
+        // Задача 2
+        dog += 4;
+        cat += 4;
+        paper += 4;
+        System.out.println("dog + 4 = " + dog);
+        System.out.println("cat + 4 = " + cat);
+        System.out.println("paper + 4 = " + paper);
+
+        // Задача 3
+        var dogFloat = dog - 3.5;
```

Comment:
здесь не нужно было заводить новую переменную. Нужно было задать значение переменной dog как 8.0 изначально, чтобы определить её тип как double и здесь тогда получилось бы сделать
```suggestion
        dog -= 3.5;
```